docs/frameworks.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  coreFrameworks: ['react', 'vue', 'angular', 'web-components'],
+  coreFrameworks: ['react', 'vue', 'angular', 'web-components', 'nextjs'],
```

Comment:
Why was this added? Doing so will result in `frontpage` adding it to the [list of core frameworks everywhere](https://github.com/storybookjs/frontpage/blob/main/site-metadata.js#L1-L5), including the FrameworkSelector menu.

Reply:
The same exact reason why Qwik was introduced. We're providing support for it. Shouldn't they be featured? if so, more than glad to update it.

Reply:
I don't understand your comment about Qwik. That's not in the list of current or proposed (by this PR) core frameworks.

I suppose, technically, adding `nextjs` as an option in the FrameworkSelector menu would just result in all snippets falling back to `react`. Which in this specific case, would be fine.

But imagine adding `sveltekit` (which is just as valid as `nextjs`, imo)... all of those snippets would also fall back to `react`, which would be incorrect.

Adding more core frameworks requires `frontpage` updates, and probably more careful consideration.

Reply:
Yes, NextJS defaults React. I've tested it locally. And I'm aware of Sveltekit and related frameworks. To keep this moving I'll remove it for now and follow up with you so that we can provide a better way to document it. I'll change it in a bit.